src/_locales/pt_BR/messages.json

```diff
@@ -0,0 +1,755 @@
+  },
+  "html_shortcuts_title": {
+    "message": "Atalhos do teclado"
+  },
+  "html_shortcuts_keyboard_shortcuts": {
+    "message": "Atalhos do teclado"
+  },
+  "html_shortcuts_context_menu_note": {
+    "message": "Nota: Essas ações também estão disponíveis através do menu de contexto do botão direito do mouse."
+  },
+  "html_shortcuts_remap_keys": {
+    "message": "Alterar atalhos"
+  },
+  "html_sidebar_general_settings": {
+    "message": "Configurações"
+  },
+  "html_sidebar_session_management": {
+    "message": "Gerenciamento de sessões"
+  },
+  "html_sidebar_shortcuts": {
+    "message": "Atalhos do teclado"
+  },
+  "html_sidebar_support": {
+    "message": "Sobre"
+  },
+  "html_success_great_success": {
+    "message": "Grande sucesso!"
+  },
+  "html_success_all_tabs_restored": {
+    "message": "Todas as suas guias foram restauradas."
+  },
+  "html_success_goto_advanced_prefix": {
+    "message": "For advanced session management, visit the"
+  },
+  "html_success_goto_advanced_suffix": {
+    "message": "página"
+  },
+  "html_session_manager": {
+    "message": "gerenciamento de sessões"
+  },
+  "html_suspended_title": {
+    "message": "Guia suspensa"
+  },
+  "html_suspended_tab_suspended": {
+    "message": "Guia suspensa"
+  },
+  "html_suspended_tab_reload": {
+    "message": "Recarregar guia"
+  },
+  "html_suspended_tab_whitelist_prefix": {
+    "message": "Nunca suspender"
+  },
+  "html_suspended_tab_whitelist_suffix": {
+    "message": "..."
+  },
+  "html_suspended_click_to_reload": {
+    "message": "Atualize ou clique para recarregar"
+  },
+  "html_suspended_toast_not_connected": {
+    "message": "Você não está conectado à Internet"
+  },
+  "html_suspended_toast_reload_disabled": {
+    "message": "O recarregamento automático de guias está temporariamente desativado."
+  },
+  "html_suspended_donation_question": {
+    "message": "Esta extensão te faz feliz?\nMostre seu apoio doando."
+  },
+  "html_permissions_title": {
+    "message": "Permissões"
+  },
+  "html_permissions_heading": {
+    "message": "Por favor, me dê acesso a URLs de arquivos locais"
+  },
+  "html_permissions_info": {
+    "message": "No momento, esta extensão não é permitida a suspender as URLs que começam com a palavra \"file\".\nPara mudar isso, ative \"Permitir acesso a URLs de arquivo\" na página de configurações da extensão."
+  },
+  "html_permissions_backup": {
+    "message": "Observação: a alteração dessa opção fará com que todas as suas guias suspensas desapareçam brevemente.\nNeste caso, agora pode ser um bom momento para fazer um backup da sua sessão de guias atuais."
+  },
+  "html_update_title": {
+    "message": "Atualização"
+  },
+  "html_update_heading2": {
+    "message": "está prestes a ficar maior"
+  },
+  "html_update_info": {
+    "message": "Uma nova versão do Grande Suspensor está pronta para ser instalada.\nObservação: durante a atualização, suas guias suspensas desaparecerão brevemente."
```

Comment:
"message": "Uma nova versão do The Great Suspender está pronta para ser instalada.\nObservação: durante a atualização, suas guias suspensas desaparecerão brevemente."